src/test/scala/com/fulcrumgenomics/vcf/filtration/ReadEndSomaticVariantFilterTest.scala

```diff
@@ -42,15 +43,153 @@ class EndRepairArtifactLikelihoodFilterTest extends UnitSpec {
     Genotype(as, "s1", as.toIndexedSeq, phased=false)
   }
 
-  "appliesTo" should "return false for any event that is not a SNP" in {
-    val filter  = new EndRepairArtifactLikelihoodFilter()
+  "isSnv" should "return true for any SNV" in {
+    isSnv(singleGenotype("A",  "C")) shouldBe true
```

Comment:
Can test like this:

```scala
val bases = Seq("A", "C", "G", "T")
for (ref <- bases; alt <- bases) yield isSnv(singleGenotype(ref, alt)) shouldBe true
```

A for-comprehension is basically a bunch of 'maps' in which you have access to all the LHS (left-hand-side) variables. What is does is the Cartesian product over all the iterables so you get all pairwise combinations. In the above example, every pairwise combo is emitted, however, you can include guards if you wanted like this (skipping all the no-variant pairs):

```scala
val bases = Seq("A", "C", "G", "T")
for (ref <- bases; alt <- bases; if (ref != alt)) yield isSnv(singleGenotype(ref, alt)) shouldBe true